tests/integration_test/test_internal.yml

```diff
@@ -0,0 +1,42 @@
+poc: ../../nvflare/poc
```

Comment:
file name test_internal.py  ==> internal_test.py

Reply:
This YAML file is specifying a fixture/config for testing though.
But I don't know what name is better.

Reply:
I see,  this is not the xxx_test.py.  then that's ok. Just looking for consistency